src/linux/system.rs

```diff
@@ -246,6 +247,7 @@ impl System {
                         parts.next().map(|v| to_u64(v)).unwrap_or(0),
                         parts.next().map(|v| to_u64(v)).unwrap_or(0),
                     );
+                    self.processors[i].frequency = get_cpu_frequency(i);
```

Comment:
I don't think this is needed: the frequency won't change.

Reply:
The frequency will change since  `get_cpu_frequency(i)` returns the current frequency. I tested this on my Raspberry Pi and depending on the load, the frequency changed. Am I missing something here?

Reply:
I didn't know that the frequency could change... Well, TIL. :)